lib/cookie.js

```diff
@@ -603,13 +619,13 @@ Cookie.prototype._initialCreationTime = null; // Used to determine if cookie.cre
 Cookie.prototype._creationRuntimeIdx = null; // Runtime index of the created cookie, used in cookieCompare()
 Cookie.prototype.lastAccessed = null; // Date when set
 
-var cookieProperties = Object.freeze(Object.keys(Cookie.prototype).map(function(p) {
-  if (p instanceof Function) {
-    return;
-  }
-  return p;
-}));
-var numCookieProperties = cookieProperties.length;
+Cookie.serializableProperties = Object.keys(Cookie.prototype)
```

Comment:
I think it's OK to have this non-frozen, documented, and therefore customizable. What do yall think?